benchmarks/integerAddition.ts

```diff
@@ -6,7 +6,6 @@ import type * as RDF from '@rdfjs/types';
 import type { Event } from 'benchmark';
 import { Suite } from 'benchmark';
 import * as Benchmark from 'benchmark';
-import * as LRUCache from 'lru-cache';
```

Comment:
Can it be removed from our package.json as well?

Reply:
Good catch. Seems like it, yes!

Reply:
Woeps, completly forgot that the openWorld type discoverer also uses the LRUCache. I'll add it back in.